cmd/util/ledger/migrations/cadence_values_migration_test.go

```diff
@@ -2733,7 +2733,7 @@ func TestTypeRequirementRemovalMigration(t *testing.T) {
 
 	contractName := "TiblesProducer"
 
-	// Store a value with the actual `TiblesProducer.Minter` type
+	// Store a value with the actual [TiblesProducer.Minter] type
```

Comment:
This is referencing a Cadence smart contract, so the `[]` won't resolve as a link.